### PR TITLE
Check user capabilities for apps

### PR DESCRIPTION
**Pull request by ys**
This change set allow us to check if a user has view capability for an app.

## How?

```
user = Escobar::Heroku::User.new(client)
user.can_view_app?(app_id) # => true or false
```

### CODE REVIEW

**Comment by ys** [comment]
[[slash-heroku] Create helpers to check capabilities for pipelines and users](https://trello.com/c/LjdfO4lk/2775-slash-heroku-create-helpers-to-check-capabilities-for-pipelines-and-users)

**Comment by tbalthazar** [comment]
+1

File: lib/escobar/heroku/user.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        can?("app", app_id, "view")
+      end
+
+      def can?(resource_type, resource_id, capability)
+        body = {
+          resource_type: resource_type,
+          resource_id: resource_id,
+          capability: capability
+        }
+        response = client.put("/users/~/capabilities",
```
Comment (atmos) [inline lib/escobar/heroku/user.rb]: How is this different than a normal `get("/apps/:app_uuid")`

Comment (ys) [inline lib/escobar/heroku/user.rb]: It never 404 I think or raise error and you can check multiple apps at once if you want too, and check for `:view` or `:deploy`. but in this specific case, unsure :D

**Pull request merged**